utils/sync-results.sh

```diff
@@ -11,6 +11,7 @@ cd ..
 # location for the shared data
 share_base=/shared/data
 modules_base=${share_base}/scpca-downstream-analyses
+s3_base=s3://scpca-downstream-analyses
```

Comment:
```suggestion
s3_base=s3://scpca-references/example-data/scpca-downstream-analyses
```
Let's actually put these in our public bucket for scpca where we already have stored example results files. I think this makes more sense instead of starting a new bucket.